dhis-2/dhis-test-e2e/README.md

```diff
@@ -23,6 +32,25 @@ mvn test \
     -Dtest.track_called_endpoints=true
 ```
 
+### From IntelliJ
+
+Make sure the property instance.url in `dhis-2/dhis-test-e2e/src/main/resources/config.properties`
+matches the url configured in your local tomcat installation. Then add `/api` at the end.
+
+For instance, if the Tomcat run/debug config has the url: `http://localhost:8080/`, then the
+instance.url property needs to be: `http://localhost:8080/api`
+
+Run Tomcat and when it's up and running, run the e2e test(s) you are interested in.
+
+#### Troubleshooting
+
+You might get errors while importing data during the test setup, it's quite possibly related to the
+database you are using.
+
+Until we find a better solution, and only if the error code is `E5002`, you can skip the reference
+check in `ReferencesCheck.checkReference` by replacing the following
+condition `if (!(isUserReference && skipSharing))` to `if (false)`
```

Comment:
I wouldn't advise this in a README file.
If we put it here, it will stay here for ever.

Reply:
I run tests against the dockerized setup so the DB is empty. Is the issue not that you might have data in your DB that is causing the problems? So the suggestion should rather be that you use a clean DB which is generally a good idea when running tests.